vhdbuilder/packer/test/run-test.sh

```diff
@@ -109,13 +110,22 @@ if [ "$OS_TYPE" == "Linux" ]; then
     exit 1
   fi
 else
+  SCRIPT_PATH="$CDIR/../$WIN_CONFIGURATION_SCRIPT_PATH"
+  echo "Run $SCRIPT_PATH"
+  az vm run-command invoke --command-id RunPowerShellScript \
+    --name $VM_NAME \
+    --resource-group $RESOURCE_GROUP_NAME \
+    --scripts @$SCRIPT_PATH \
+    --output json
+
   SCRIPT_PATH="$CDIR/$WIN_SCRIPT_PATH"
+  echo "Run $SCRIPT_PATH"
```

Comment:
combine the test to take less time?

Reply:
otherwise, how do we handle the first run-command error?

Reply:
I did not find a way to run two scripts at the same time in `az vm run-command`.
The first step normally is not thrown errors. If it has some issue, we can capture the errors when running the second script.